### PR TITLE
Cn downstream

### PR DESCRIPTION
**Pull request by leseb**

### CODE REVIEW

File: cmd/cluster.go [hunk 1]
```diff
@@ -1,3 +1,25 @@
+/*
+ * Ceph Nano (C) 2017 Red Hat, Inc.
```
Comment (ErwanAliasr1) [inline cmd/cluster.go]: 2018 :)

File: cmd/image.go [hunk 2]
```diff
@@ -1,3 +1,25 @@
+/*
+ * Ceph Nano (C) 2017 Red Hat, Inc.
```
Comment (ErwanAliasr1) [inline cmd/image.go]: Ditto

File: cmd/image_list.go [hunk 3]
```diff
@@ -1,3 +1,25 @@
+/*
+ * Ceph Nano (C) 2017 Red Hat, Inc.
```
Comment (ErwanAliasr1) [inline cmd/image_list.go]: Ditto.

File: cmd/image_update.go [hunk 4]
```diff
@@ -1,3 +1,25 @@
+/*
+ * Ceph Nano (C) 2017 Red Hat, Inc.
```
Comment (ErwanAliasr1) [inline cmd/image_update.go]: Ditto

File: cmd/kube.go [hunk 5]
```diff
@@ -1,3 +1,25 @@
+/*
+ * Ceph Nano (C) 2017 Red Hat, Inc.
```
Comment (ErwanAliasr1) [inline cmd/kube.go:2]: etc... ;)

File: cmd/main.go [hunk 6]
```diff
@@ -1,3 +1,25 @@
+/*
+ * Ceph Nano (C) 2017 Red Hat, Inc.
```
Comment (ErwanAliasr1) [inline cmd/main.go]: One more

File: cmd/purge.go [hunk 7]
```diff
@@ -1,3 +1,25 @@
+/*
+ * Ceph Nano (C) 2017 Red Hat, Inc.
```
Comment (ErwanAliasr1) [inline cmd/purge.go]: Oh !

File: cmd/stop.go [hunk 8]
```diff
@@ -1,3 +1,25 @@
+/*
+ * Ceph Nano (C) 2017 Red Hat, Inc.
```
Comment (ErwanAliasr1) [inline cmd/stop.go]: +1

**Review by ErwanAliasr1 — CHANGES_REQUESTED** [review]

File: cmd/image_list.go [hunk 9]
```diff
@@ -46,5 +48,10 @@ func CliImageList() *cobra.Command {
 
 // listImageTags lists container image tags
 func listImageTags(cmd *cobra.Command, args []string) {
-	listDockerRegistryImageTags()
+	envRegistry := os.Getenv("CN_REGISTRY")
```
Comment (ErwanAliasr1) [inline cmd/image_list.go]: We may avoid create a variable if that is used once

File: cmd/main.go [hunk 10]
```diff
@@ -130,6 +130,11 @@ func Main(version string) {
 }
 
 func init() {
+	envRegistry := os.Getenv("CN_REGISTRY")
```
Comment (ErwanAliasr1) [inline cmd/main.go]: ditto

**Review by ErwanAliasr1 — CHANGES_REQUESTED** [review]

**Comment by leseb** [comment]
@ErwanAliasr1 comments have been addressed

**Review by ErwanAliasr1 — APPROVED** [review]

**Pull request merged**